### ISSUE TITLE
solution for escape key in macros

### ISSUE THREAD

**Issue by sunaku** [vim-utils/vim-alt-mappings#5]
@bruno- I solved this "ESC breaks macros" issue long ago!  See:
https://github.com/sunaku/.vim/blob/config/plugin/escape.vim

**Comment by bruno-** [comment]
Hey,
interesting. Are there any downsides to that solution?

**Comment by sunaku** [comment]
Yes, it prevents you from typing accented Latin characters in insert mode using Alt+key combos. :scream:

However, you might be able to use [the bottom part of my solution](https://github.com/sunaku/.vim/blob/config/plugin/escape.vim#L82-L108), which only deals with macros. :smile_cat:
